.rubocop.yml

```diff
@@ -1,1892 +1,87 @@
-  Enabled: false
-  Exclude: []
-  ExpectMatchingDefinition: false
-  Regex: 
-  IgnoreExecutableScripts: true
-
-Naming/MethodName:
-  Description: Use the configured style when naming methods.
-  StyleGuide: https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars
-  Enabled: false
-  EnforcedStyle: snake_case
-  SupportedStyles:
-  - snake_case
-  - camelCase
+  # We probably want to bring this down but let's start here for now
+  Max: 20
+  Exclude:
+    - "test/**/*"
 
 Naming/PredicateName:
-  Description: Check the names of predicate methods.
-  StyleGuide: https://github.com/bbatsov/ruby-style-guide#bool-methods-qmark
   Enabled: false
-  NamePrefix:
-  - is_
-  - has_
-  - have_
-  NamePrefixBlacklist:
-  - is_
-  - has_
-  - have_
-  NameWhitelist:
-  - is_a?
 
-Naming/VariableName:
-  Description: Use the configured style when naming variables.
-  StyleGuide: https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars
+Style/ConditionalAssignment:
   Enabled: false
-  EnforcedStyle: snake_case
-  SupportedStyles:
-  - snake_case
-  - camelCase
 
-Naming/VariableNumber:
+Style/StringLiterals:
   Enabled: false
 
-# Type 'Performance' (21):
-# Supports --auto-correct
-Performance/CaseWhenSplat:
-  Description: Place `when` conditions that use splat at the end of the list of `when`
-    branches.
+Style/FrozenStringLiteralComment:
   Enabled: false
 
-# Supports --auto-correct
-Performance/Casecmp:
-  Description: Use `casecmp` rather than `downcase ==`.
-  Reference: https://github.com/JuanitoFatas/fast-ruby#stringcasecmp-vs-stringdowncase---code
+Style/Documentation:
   Enabled: false
 
-# Supports --auto-correct
-Performance/Count:
-  Description: Use `count` instead of `select...size`, `reject...size`, `select...count`,
-    `reject...count`, `select...length`, and `reject...length`.
+Style/DocumentationMethod:
   Enabled: false
 
-# Supports --auto-correct
-Performance/Detect:
-  Description: Use `detect` instead of `select.first`, `find_all.first`, `select.last`,
-    and `find_all.last`.
-  Reference: https://github.com/JuanitoFatas/fast-ruby#enumerabledetect-vs-enumerableselectfirst-code
-  Enabled: false
+Naming/VariableNumber:
+  EnforcedStyle: normalcase
 
-Performance/DoubleStartEndWith:
-  Description: Use `str.{start,end}_with?(x, ..., y, ...)` instead of `str.{start,end}_with?(x,
-    ...) || str.{start,end}_with?(y, ...)`.
-  Enabled: false
+Style/LambdaCall:
+  EnforcedStyle: braces
 
-# Supports --auto-correct
-Performance/EndWith:
-  Description: Use `end_with?` instead of a regex match anchored to the end of a string.
-  Reference: https://github.com/JuanitoFatas/fast-ruby#stringmatch-vs-stringstart_withstringend_with-code-start-code-end
+Style/BlockComments:
   Enabled: false
 
-Performance/FixedSize:
-  Description: Do not compute the size of statically sized objects except in constants
+# We use Heredoc indentation in exercises
+# where variable indentiation is desirable
```

Comment:
If this is not universally (in the Ruby track) the case, then this should be enabled, and a directive comment can communicate that it is intentional.

Reply:
Happy to enable it, but then there's a few places that would need fixing and I don't know how to fix them.

Reply:
I can enable it, see what breaks.

Reply:
Feel free to continue to push, I can work through rolling updates, not a problem.

Reply:
I'm done for now. I need your help on the remaining failures I think.